File: app/src/main/java/com/dododial/phone/RuleChecker.java
package com.dododial.phone;

import java.lang.*;

public class RuleChecker {

    public boolean allowChecker(String numString) {
        String trueNum = numString.substring(4);
        Long number = Long.parseLong(trueNum);
        if (number ==  7167102601L) {
            return false;
        } else {
            return true;
        }
    }
}

File: app/src/main/java/com/dododial/phone/RuleChecker.kt
package com.dododial.phone

import android.icu.util.UniversalTimeScale.toLong
import android.util.Log

object RuleChecker {

    fun isSafe(numberString: String?): Boolean {

        // For null conference host
        if (numberString == null) {
            return true
        }

        // Don't remember what substring was for.
        val number = numberString.toLong()
        Log.i("DODODEBUG", number.toString())

        return number != 7167102601L
    }
}
